### DISCUSSION TITLE
Where to upload the block lists to? on Windows 11

### DISCUSSION THREAD

**Discussion by ghost** [StevenBlack/hosts#2700] [Q&A]
Cold Turkey my current blocker, will not accept more than 500 entries at a time. And it slows down my operating system after 5000 entries... Where should I upload the gambling block lists to, to have them block those sites on a windows 11 operating system. Preferably password controlled with accountability partner and very hard to uninstall.

**Comment by iam-py-test** [comment]
Does your content filter add entries to the Windows HOSTs file?
If so, then Windows can not handle large HOSTs files without some configuration changes: see the README section _[Warning: Using this hosts file in Windows may require disabling DNS Cache service](https://github.com/StevenBlack/hosts?tab=readme-ov-file#warning-using-this-hosts-file-in-windows-may-require-disabling-dns-cache-service)_
If you wish to continue using large blocklists and the aforementioned fix does not work or is undesirable, you may want to use a DNS based blocker (PiHole, AdGuard Home).
Thank you
